blacken_docs.py

```diff
@@ -95,6 +99,9 @@ def _collect_error(match: Match[str]) -> Generator[None, None, None]:
             errors.append(CodeBlockError(match.start(), e))
 
     def _md_match(match: Match[str]) -> str:
+        lang = match['lang']
+        if lang is not None and lang not in MD_PY_LANGS:
+            return match[0]
```

Comment:
this is significantly slower since it'll now match this callback for every language block